### DISCUSSION TITLE
Custom domain is not working.

### DISCUSSION THREAD

**Discussion by merndev1101** [vercel/community#5143] [Help] [answered]
### Summary

I deployed my porfolio site into vercel.
I have to customize domain to yousefbakier.com.
The DNS configures are working and I waited for 48 hours.
But custom domain (yousefbakier.com) is not working.

### Steps to Reproduce

Here are the screenshots attached.
![recordconfiguration](https://github.com/vercel/community/assets/132189735/2aacb7f8-d153-4021-b1a0-d15b76616082)
![whois com](https://github.com/vercel/community/assets/132189735/4371aa8e-2c1f-4e74-a019-5ba108e08d91)
![yousefbakier com](https://github.com/vercel/community/assets/132189735/e8ba3ae3-84c7-4a00-aefb-3c38441ed9f5)
![configurations](https://github.com/vercel/community/assets/132189735/be34232c-cd61-46e6-919e-7a0588bb703d)

**Comment by swarnava** [comment, marked as answer]
Hi there,

It looks like your DNS isn't pointing to Vercel right now: 

https://dnschecker.org/#A/yousefbakier.com
https://dnschecker.org/#CNAME/www.yousefbakier.com

**Reply by merndev1101** [reply]
Thank you for the replying.
Could you tell me how to resolve this problem in detail?
